src/components/modules/crossBlockSelection.ts

```diff
@@ -44,6 +57,22 @@ export default class CrossBlockSelection extends Module {
       !!this.lastSelectedBlock;
   }
 
+  /**
+   * Enables Cross Block Selection
+   *
+   * @param {MouseEvent} event - mouse down event
+   */
+  private enableCrossBlockSelection(event: MouseEvent): void {
```

Comment:
I don't think `enableCrossBlockSelection` is a good name here but I can't come up with a better one at the moment